Agents/CEAAgent/README.md

```diff
@@ -1,97 +1,83 @@
+### 2.6. TargetresourceID for Building Geometry Query
+For building geometry queries, the CEA agent defaults to TheWorldAvatar Blazegprah. The targetResourceIDs in ```./cea-agent/src/main/resources/CEAAgentConfig.properties``` provides mappings to namespaces in TheWorldAvatar Blazegraph (http://www.theworldavatar.com:83/citieskg/). The targetResourceID are passed to the access agent in order to query from TheWorldAvatar Blazegraph. Check that a targetResourceID to pass to the access agent exists for the namespace being used for SPARQL queries.
 Currently included are:
-
 - ```citieskg-berlin```
 - ```singaporeEPSG24500```
 - ```citieskg-singaporeEPSG4326```
 - ```citieskg-kingslynnEPSG3857```
 - ```citieskg-kingslynnEPSG27700```
 - ```citieskg-pirmasensEPSG32633```
 
-If not included, you will need to add the targetResourceID in ```./cea-agent/src/main/resources/CEAAgentConfig.properties``` and add the corresponding mapping from cityObject IRI to targetResourceID in accessAgentRoutes in the ```readConfig``` method of ```./cea-agent/src/main/java/uk/ac/cam/cares/twa/cities/ceaagent/CEAAgent.java```.
+If not included, you will need to add the targetResourceID in ```./cea-agent/src/main/resources/CEAAgentConfig.properties``` and add the corresponding mapping from cityObject IRI to targetResourceID in accessAgentRoutes in the ```readConfig``` method of ```./cea-agent/src/main/java/uk/ac/cam/cares/jps/agent/cea/CEAAgent.java```.
 
-### For Developers
-In order to use a local Blazegraph, you will need to run the access agent locally and set the accessagent.properties storerouter endpoint url to your local Blazegraph, as well as add triples for your namespace to a local ontokgrouter as is explained [here](https://github.com/cambridge-cares/CitiesKG/tree/develop/agents#install-and-build-local-accessagent-for-developers). In order fo the time series client to use the local PostgreSQL and the local Blazegraph, in ```/cea-agent/src/main/resources/timeseriesclient.properties```, change ```db.url``` to the local PostgreSQL database.
-
-### Running
+### 2.7. Build and Run
+In the same directory as this README, first build the Docker image by running
+```
+docker-compose build
+```
 
-To build and start the agent, you simply need to spin up a container.
-In Visual Studio Code, ensure the Docker extension is installed, then right-click docker-compose.yml and select 'Compose Up'.
-Alternatively, from the command line, and in the same directory as this README, run
+After the image is built, copy ```./stack-manager-input-config/cea-agent.json``` and place it in ```../Deploy/stacks/dynamic/stack-manager/inputs/config/services```. Then, in the ```../Deploy/stacks/dynamic/stack-manager/``` directory, run 
 ```
-docker-compose up -d
+./stack.sh start <STACK NAME>
 ```
+Replace ```<STACK NAME>``` with the name of the stack that was spun up by Stack Manager.
+
+WARNING: tests currently not running.
```

Comment:
Please expand this to explain the two points we have discussed, namely 1) that the tests don't appear to be run when building the docker image, even though they should, and 2) the fundamental issue of the tests failing (if they were executed) in the stack.